powerdns/models/powerdns.py

```diff
@@ -627,9 +622,20 @@ def update_serial(sender, instance, **kwargs):
         soa.save()
 
 
+@receiver(post_save, sender=Domain, dispatch_uid='domain_update_ptr')
+def update_ptr(sender, instance, **kwargs):
```

Comment:
as far as i understand, this signal handler should update ptrs if domain has changed. Coulnd't we check if this option on domain has changed during current save? Otherwise, for "occupied" domains we'll trigger hundreds or thousands of records saves.